packages/tslint-rules/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "office-ui-fabric-react-tslint",
+  "name": "@uifabric/tslint-rules",
   "version": "6.0.0",
   "description": "Shared tslint rules",
```

Comment:
I feel like leaf dependencies like this which will rarely change should not be tied to Fabric version. They should adhere to semver particularly if they're meant to be used standalone in other repos.

Reply:
I've moved it to 1.0.0